pkgs/servers/udpt/default.nix

```diff
@@ -21,7 +21,6 @@ rustPlatform.buildRustPackage rec {
     description = "A lightweight UDP torrent tracker";
     homepage = "https://naim94a.github.io/udpt";
     license = lib.licenses.gpl3;
-    platforms = lib.platforms.linux;
```

Comment:
I think we should set this to platforms.all instead.

Reply:
Done.